com.skype.Client.json

```diff
@@ -15,9 +15,12 @@
         "--share=network",
         "--system-talk-name=org.freedesktop.NetworkManager",
         "--system-talk-name=org.bluez",
+        "--filesystem=xdg-documents",
         "--system-talk-name=org.freedesktop.login1",
         "--filesystem=xdg-download",
-        "--filesystem=home:ro",
+        "--filesystem=xdg-music",
+        "--filesystem=xdg-pictures",
+        "--filesystem=xdg-videos",
```

Comment:
I suggest making everything `--filesystem=xdg-*:ro`, except `xdg-download`, since the current permissions are ro.